GitUI/CommandsDialogs/FormCheckoutBranch.cs

```diff
@@ -385,21 +384,21 @@ private DialogResult PerformCheckout(IWin32Window? owner)
                     bool? messageBoxResult = AppSettings.AutoPopStashAfterCheckoutBranch;
                     if (messageBoxResult is null)
                     {
-                        using Microsoft.WindowsAPICodePack.Dialogs.TaskDialog dialog = new()
+                        TaskDialogPage page = new()
                         {
-                            OwnerWindowHandle = Handle,
                             Text = _applyStashedItemsAgain.Text,
                             Caption = _applyStashedItemsAgainCaption.Text,
-                            Icon = TaskDialogStandardIcon.Information,
-                            StandardButtons = TaskDialogStandardButtons.Yes | TaskDialogStandardButtons.No,
-                            FooterCheckBoxText = _dontShowAgain.Text,
-                            FooterIcon = TaskDialogStandardIcon.Information,
-                            StartupLocation = Microsoft.WindowsAPICodePack.Dialogs.TaskDialogStartupLocation.CenterOwner
```

Comment:
`StartupLocation` is not specified in the update. Is this the default?

Reply:
Yes, `CenterOwner` is the default location